x/tx/signing/context.go

```diff
@@ -14,6 +14,11 @@ import (
 	"cosmossdk.io/core/address"
 )
 
+type TypeResolver interface {
```

Comment:
should we put this in core and import it everywhere so we dont have the same issue in the future

Reply:
I am uncertain if its worth adding an extra package in core for that. Not sure how we would even name it.
However, if you have a proposition, I am not against it.